test/testdata/infer/fuzz_sealed.rb

```diff
@@ -0,0 +1,17 @@
+# typed: true
+
+extend T::Sig
+
+module Boolean
+  sealed!
```

Comment:
```suggestion
  extend T::Helpers
  sealed!
```

Was this meant to raise an error?

Reply:
I took this verbatim from @azdavis's report.

It's possible that missing the `extend T::Helpers` was precisely the problem?

Reply:
Ah cool, thanks!